### DISCUSSION TITLE
Kaspersky

### DISCUSSION THREAD

**Discussion by Indirectelex** [qdm12/gluetun#962] [Q&A]
Is there a way with the custom provider option to allow Kaspersky, I know they have an OpenVPN credential file, any thoughts?

**Comment by andry81** [comment]
Just for instance, they block Tor traffic and might will block other VPNs.

**Comment by qdm12** [comment]
@Indirectelex what do you mean by `allow Kaspersky`? Do you mean to support Kaspersky's Openvpn configuration file?

**Reply by Indirectelex** [reply]
I wonder if Kaspersky is compatible with gluetun so I can allow it on docker-compose. using Kaspersky's Openvpn configuration file

**Comment by qdm12** [comment]
@Indirectelex Follow the steps at https://github.com/qdm12/gluetun/wiki/OpenVPN-configuration-file

**Reply by Indirectelex** [reply]
So basically you are saying as soon as they get a Openvpn configuration file then it should work? Ok thanks Ill give it a try

**Comment by Indirectelex** [comment]
![WindowsTerminal_TN4SFUh1mF](https://user-images.githubusercontent.com/41977588/166616562-78a90ca5-1afb-4847-b600-e4ef5fa9c23e.png)

**Reply by qdm12** [reply]
Check your bind mount path, you mounted it as `/gluetun.custom.conf:ro` instead of `/gluetun/custom.conf:ro`

**Comment by Indirectelex** [comment]
Kaspersky have a .ovpn file and I dont see where to link it to the container

**Reply by Rohaq** [reply]
Are you able to post the OpenVPN file?

**Reply by Indirectelex** [reply]
[credentials.zip](https://github.com/qdm12/gluetun/files/8622279/credentials.zip)
Am doing it for science, I hope for the best

**Reply by qdm12** [reply]
See comment above, you wrongly bind mounted the configuration.
Also as per the instructions as well:

- `VPN_SERVICE_PROVIDER=custom`, `kaspersky` is not known by gluetun
- you'll need to replace `location-pub-kl--us-new-york.aura-servers.com` with an IP address (i..e use `nslookup location-pub-kl--us-new-york.aura-servers.com`)
- it may complain about the two `ca` certificates instead of one. In this case you can 'quick-fix' it by just having one of them, but please create an issue and I'll take care of multiple certificates reading for custom configs.

Also careful sharing your openvpn user and password in the screenshot above, you might want to renew them.
